glTF-Toolkit/src/GLTFLODUtils.cpp

```diff
@@ -375,7 +386,7 @@ GLTFDocument GLTFLODUtils::MergeDocumentsAsLODs(const std::vector<GLTFDocument>&
 
     for (size_t i = 1; i < docs.size(); i++)
     {
-        gltfPrimary = AddGLTFNodeLOD(gltfPrimary, lods, docs[i]);
+        gltfPrimary = AddGLTFNodeLOD(gltfPrimary, lods, docs[i], relativePaths[i]);

```

Comment:
the first element of relativePaths is always ignored. Maybe the input should be only the non-lod0 relative paths?

Reply:
true, I'll make it so